tools/gfx/vulkan/render-vk.cpp

```diff
@@ -7956,6 +7958,78 @@ Result VKDevice::createTextureView(ITextureResource* texture, IResourceView::Des
     return SLANG_OK;
 }
 
+// TODO: Buffer vs texture specific?
```

Comment:
We only care about images at this point.
We need to make sure we return correct bits for ShaderResource, RenderTarget, UnorderedAccess, DepthWrite, Present and TransferSrc/TransferDest states.